lib/generators/draper/templates/application_decorator.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationDecorator < Draper::Decorator
```

Comment:
I agree completely. It's best not to assume functionality at the application level. I'd rather the user add this line themselves that way they are forced to understand what it does.